test/integration/v3/bundles/css.test.js

```diff
@@ -305,5 +306,67 @@ describe("/v3/bundles/css", function() {
 					});
 			});
 		});
+
+		context(
+			"works with all active and maintained components which have no defined brand support",
+			function() {
+				const components = require("../../../../scripts/components.json");
+				const mappedComponents = components.filter(component => {
+					return component.languages.includes("scss");
+				});
+				for (const component of mappedComponents) {
+					if (component.brands.length === 0) {
+						it(`/v3/bundles/css?modules=${
+							component.name
+							// eslint-disable-next-line no-loop-func
+						}@${component.version}&source=test`, function() {
+							return request(app)
+								.get(`/v3/bundles/css?modules=${component.name}&source=test`)
+								.expect(200)
+								.expect("Content-Type", "text/css; charset=utf-8");
+						});
+					}
+				}
+			},
+		);
+
+		context(
+			"works with all active and maintained components which have defined brand support",
+			function() {
+				const components = require("../../../../scripts/components.json");
+				const mappedComponents = components.filter(component => {
+					return component.languages.includes("scss");
+				});
+				for (const component of mappedComponents) {
+					if (component.brands.length > 0) {
+						for (const brand of component.brands) {
+							it(`/v3/bundles/css?modules=${component.name}@${
+								component.version
+								// eslint-disable-next-line no-loop-func
+							}&source=test&brand=${brand}`, function() {
```

Comment:
could maybe but this test string building in a separate function, it gets kinda hard to read broken out into these lines.